export/distro/registrations.go

```diff
@@ -51,7 +51,7 @@ func (reg *registrationInfo) update(newReg export.Registration) bool {
 	case export.FormatSerialized:
 		// TODO reg.format = distro.NewSerializedFormat()
 	case export.FormatIoTCoreJSON:
-		// TODO reg.format = distro.NewIotCoreFormat()
+		reg.format = jsonFormatter{}
```

Comment:
I know it has already merged, but FormatIotCoreJSON could be removed. It make no sense to have two format types that generate the same output.

Reply:
@feclare you can send the fix, especially if demands refactoring of `distro`. I would rather redirect Mainflux resources to other connectors ASAP.